backend/README.md

```diff
@@ -0,0 +1,41 @@
+Backend
+
+The backend server is built with Express.js and Node.js, which provides completed REST APIs for data interaction.
+
+- Node.js
+node.js installed from http://nodejs.org
+
+- Database
+MongoDB atlas is used as the backend database.
+
+Mongoose is used to access the MongoDB for post, get, put and delete.
+
+PORT: http://localhost:5000
+
+RESTful API between backend and web/mobile is documented
+// https://app.swaggerhub.com/apis/Kodstar/Issue_Tracker_3/1.0.1
+
+
+- Usage
+
+Running locally you need 2 terminals open: one for server, and another one for MongoDB backend. 
+
+-Install node.js; 
+
+-git clone master branch
+
+Go to directory issue-tracker-2020-3/backend, and run npm install;
+
+-installed packages
+* express
+* mongoose
+* cors
+* nodemon
+* dotenv
+* eslint;
+* .env file added in backend directory. 
+(The .env file is in the files of teams group-3.)
```

Comment:
Instead of referring to private file in Teams group, also describe the variables that needs to be defined in .env file so that someone else can also install and run the application with specifying his/her own database url etc.